packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -209,7 +209,9 @@ export default async args => {
             commonjs({
                 extensions: config.extensions,
                 transformMixedEsModules: true,
-                requireReturnsDefault: true,
+                requireReturnsDefault: id => {
+                    return !id.includes("react-lifecycles-compat");
```

Comment:
This one seems to me that is really specific for the image-viewer, which leads me to request you to create a custom config for it 😅